sheet/jarjee/JeanClaude.qml

```diff
@@ -374,11 +390,6 @@ Item {
                 Bonus {
                     name: "Touch Penalty"
                     amount: -acArmourBonus.amount
-                },
-                Bonus {
-                    source: weararmour
```

Comment:
With all my other bonus sources all the way at the bottom of the character sheet. I'll make all the names camelCase.